gradle/libs.versions.toml

```diff
@@ -1,14 +1,14 @@
 [versions]
-plugin-android = "8.0.2"
-plugin-kotlin = "1.8.21"
+plugin-android = "8.1.0-rc01"
```

Comment:
do we want to use not stable versions?

Reply:
is it necessary?

Reply:
It was necessary yes, there were a couple of things that needed to be upgraded together. We can however wait until the Android plugin is stable?

Reply:
@terrakok You can re-review this comment, it's been updated